tests/commands/test_changelog_command.py

```diff
@@ -676,12 +688,13 @@ def test_changelog_from_rev_single_version_not_found(
     with pytest.raises(NoCommitsFoundError) as excinfo:
         cli.main()
 
-    assert "No commits found" in str(excinfo)
+    assert "Could not find a valid revision" in str(excinfo)
```

Comment:
Do we need to use the approach above as well?

Reply:
In this case `commitizen/changelog.py` is what raised the exception message (in English) so no need to have `LC_ALL`:

![image](https://user-images.githubusercontent.com/33836927/183441489-ec1402bd-cfa0-4d36-89c9-9987a5cc0b62.png)

Reply:
ah got it. Thanks @bhelgs !